src/HTCondor/condor-9-to-10-upgrade-check.py

```diff
@@ -56,38 +75,42 @@ def getTokenConfiguration(param, is_ce, checking_remote_collector):
                     break
     if not ret_info["using_token_auth"]:
         return ret_info
-    #Unexpanded default values for TRUST_DOMAIN
+    # Unexpanded default values for TRUST_DOMAIN
     default_trust_domain = "$(COLLECTOR_HOST)"
     if ret_info["version"] >= 10:
         default_trust_domain = "$(UID_DOMAIN)"
-    #Create command line to run to get unexpanded TRUST_DOMAIN value from config
+    # Create command line to run to get unexpanded TRUST_DOMAIN value from config
     cmd = []
     if is_ce:
         cmd.append("condor_ce_config_val")
     else:
         cmd.append("condor_config_val")
     if checking_remote_collector:
         cmd.append("-collector")
-    cmd.extend(["-dump","TRUST_DOMAIN"])
-    p = subprocess.run(cmd,stdout=subprocess.PIPE)
+    cmd.extend(["-dump", "TRUST_DOMAIN"])
```

Comment:
`listA.extend(listB)` is a new one for me, I think you can also do `listA += listB`, but this is fine.

Reply:
A little digging and can confirm that listA += listB does also work. However, I will keep this for the time being